src/mame/bmc/koftball.cpp

```diff
@@ -143,58 +142,13 @@ TILE_GET_INFO_MEMBER(koftball_state::get_tile_info)
 	tileinfo.set(0, data, 0, 0);
 }
 
-// TODO: the following 'pixpal' methods were copied from bmc/bmcpokr.cpp. They need adapting.
-
-void koftball_state::pixram_w(offs_t offset, uint16_t data, uint16_t mem_mask)
-{
-	COMBINE_DATA(&m_pixram[offset]);
-
-	int const x = (offset & 0xff) << 2;
-	int const y = (offset >> 8);
-
-	uint16_t const pixpal = (m_pixpal & 0xf) << 4;
-
-	uint16_t pen;
-	if (ACCESSING_BITS_8_15)
-	{
-		pen = (data >> 12) & 0xf; m_pixbitmap->pix(y, x + 0) = pen ? pixpal + pen : 0;
-		pen = (data >>  8) & 0xf; m_pixbitmap->pix(y, x + 1) = pen ? pixpal + pen : 0;
-	}
-	if (ACCESSING_BITS_0_7)
-	{
-		pen = (data >>  4) & 0xf; m_pixbitmap->pix(y, x + 2) = pen ? pixpal + pen : 0;
-		pen = (data >>  0) & 0xf; m_pixbitmap->pix(y, x + 3) = pen ? pixpal + pen : 0;
-	}
-}
```

Comment:
I got rid of this because it's hardly adaptable, the performance penalty for drawing 512x256 in screen_update rather than the handler isn't all that noticeable in a stock 68k game.
It's also imho the wrong way of doing it, if anybody really bothers the better approach is to just use a `is_dirty` boolean flag attached to the handlers.